### DISCUSSION TITLE
Installation of Linkerd2

### DISCUSSION THREAD

**Discussion by DaspawnW** [linkerd/linkerd2#5303] [Q&A] [answered]
Dear Community,

there are two ways how to install Linkerd2:
* Helm Chart
* linkerd-CLI

Out of experience and what happened in the past with Istio (for sure Linkerd2 is here hopefully completely different) they changed 2 up to 3 times the way how to install istio:
* helm chart
* ansible chart
* operator
* what ever is next

When I read the documentation of Linkerd2 the documentation favours the cli, but for most of the articles there is also a helm installation instruction.

Is there a recommended way how to install or maybe a drawback you get with installation via cli compared to installation helm or vice versa?

Thank you in advance,
Best regards,
 Björn

**Comment by alpeb** [comment, marked as answer]
Hi Björn,

That's a great question. The goal of the CLI is to provide the easiest experience to onboard new users. But once you decide to go into prod, the recommended way is using the Helm chart, as it provides you with complete configurability, adapts well CI pipelines, and integrates well with other tools. That being said, there are folks that have been successful as well just relying on the CLI in prod :wink:

**Reply by DaspawnW** [reply]
thnx for this information :-)
